### ISSUE TITLE
DefaultMunch getter now returns None instead of DefaultValue

### ISSUE THREAD

**Issue by Canuteson** [Infinidat/munch#53]
If a DefaultMunch has a default value of None, munch.get(attr_name, default_value) will now return None instead of default_value

```
>>> from munch import DefaultMunch
>>> d = DefaultMunch(None, {'a': 1})
>>> d.get('a')
1
>>> d.get('b')
>>> d.get('b', 'test')
>>> d.get('b', 'test') is None
True
```

This is a devastating change in behavior from the previous version:
```
>>> from munch import DefaultMunch
>>> d = DefaultMunch(None, {'a': 1})
>>> d.get('a')
1
>>> d.get('b')
>>> d.get('b', 'test')
'test'
>>> d.get('b', 'test') is None
False
```

**Comment by ayalash** [comment]
@Canuteson Thanks for reporting.
A bug fix version (2.4.1) will be released tomorrow.

**Comment by Canuteson** [comment]
Thanks for the quick turn around on this!
